Fix Registration/Login

VianneyPham opened the pull request:
I wanna see if I'm committing correctly

VianneyPham committed:
Fix Registration/Login

I wanna see if I'm committing correctly

maddy-dc-minnstate commented:
This is perfect. :) Now feel free to update this branch with more changes as you go.

VianneyPham committed:
Update admin.php

Made sure people can't URL snipe the admin.php site without having role:"admin"

VianneyPham committed:
fixed account registration

VianneyPham committed:
Fixed session bug in header.php

would always show a warning message about having an active session already

VianneyPham committed:
Update header.php
